build.sh

```diff
@@ -813,6 +825,7 @@ case "$COMMAND" in
 
     "verify-osx-object-server")
         sh build.sh download-object-server
+        sh build.sh reset-object-server
```

Comment:
I think this should be moved into `+[RLMSyncTestCase setUp]` because otherwise you won't kill previously running servers if you're running these tests via Xcode.